modules/userId/index.js

```diff
@@ -319,15 +352,7 @@ function getStoredValue(submodule, key = undefined) {
     if (storage.type === COOKIE) {
       storedValue = mgr.getCookie(storedKey);
```

Comment:
Optional: Even though it's just one line, it might be a good idea to extract the `mgr.getCookie(storedKey)` into a function `getValueFromCookie` in order to keep the abstraction level consistent. It could make further refactoring opportunities more apparent later on.